docs/use-cases/one-to-many.md

```diff
@@ -0,0 +1,138 @@
+
+{{<imgproc src="/use-cases/one-to-many/config.png" resize="800x" class="fill aligncenter" style="max-width: 400px" declaredimensions=true alt="Configuration builder UI">}}
+
+{{% /tablestep %}}
+{{% tablestep %}}
+**2. Copy the raw JSON**
+
+In your machine's **CONFIGURE** tab, switch to **JSON** and copy the raw JSON.
+
+{{<imgproc src="/use-cases/one-to-many/raw-json.png" resize="700x" class="fill aligncenter" style="max-width: 400px" declaredimensions=true alt="JSON subtab of the CONFIGURE tab">}}
+
+{{% /tablestep %}}
+{{% tablestep link="/fleet/fragments/" %}}
+**3. Create a fragment**
+
+Go to [app.viam.com/fragments](https://app.viam.com/fragments).
+
+Add a fragment, and paste the copied JSON configuration into it.
+
+Set your privacy settings.
+There are three options for this:
+
+- **Public:** Any user inside or outside of your organization will be able to view and use this fragment.
+- **Private:** No user outside of your organization will be able to view or use this fragment.
+- **Unlisted:** Any user inside or outside of your organization, with a direct link, will be able to view and use this fragment.
+
+Click **Save**.
+
+If you want to edit the fragment later, do it from this screen.
+
+{{<imgproc src="/use-cases/one-to-many/new-fragment.png" resize="700x" class="fill aligncenter" style="max-width: 350px" declaredimensions=true alt="app.viam.com/fragment interface">}}
+
+{{% /tablestep %}}
+{{% tablestep %}}
+{{<imgproc src="/use-cases/one-to-many/trash-can.svg" class="fill alignleft" style="max-width: 50px" declaredimensions=true alt="Delete">}}
+**3. Delete the original configuration (optional)**
+
+Now that the configuration is saved as a fragment, you can delete the original config from your machine and _replace it with the fragment_.
+In this way, you can keep all your machines up to date whenever you change the fragment.
+
+{{% /tablestep %}}
+{{< /table >}}
+
+## Add a fragment to multiple machines
+
+Now that you have a fragment, you can add it to as many machines as you'd like:
+
+{{< table >}}
+{{% tablestep %}}
+{{<imgproc src="appendix/try-viam/rover-resources/fragments/fragments_list.png" resize="800x" class="fill alignleft" style="max-width: 300px" declaredimensions=true alt="Add fragment">}}
+**1. Add the fragment to one machine**
+
+On your machine's **CONFIGURE** tab, click the **+** button and select **Insert fragment**.
+Search for your fragment and add it.
+
+Click **Save** in the upper right corner of the screen.
+
+{{% /tablestep %}}
+{{% tablestep %}}
+{{<imgproc src="/use-cases/one-to-many/repeat.svg" class="fill alignleft" style="max-width: 120px"  declaredimensions=true alt="Repeat">}}
+**2. Repeat for each machine**
+
+Repeat step 1 for each of the machines that you want to configure in the same way.
+
+If some of your machines have slight differences, you can still add the fragment and then add fragment mods in the next section.
+
+{{% /tablestep %}}
+{{< /table >}}
+
+## Modify a fragment
```

Comment:
```suggestion
## (Optional) Modify a fragment
```

Reply:
Why? This is a separate section, available if they want it. On https://docs.viam.com/use-cases/collect-sensor-data/ we don't list "view sensor data" as optional. I think it could get messy if we start adding optional to all sections.

Would it help if I change it to this?
```suggestion
## How to modify a fragment
```